sotodlib/core/axisman_io.py

```diff
@@ -23,6 +23,14 @@ def flatten_RangesMatrix(rm):
       Dict with arrays called 'shape', 'intervals', and 'ends'.
     """
     shape = rm.shape
+    if len(shape) == 1:
+        intervals = rm.ranges().reshape(-1)
+        ends = [len(intervals)]
```

Comment:
For consistency, "ends" should be an array of int (like in other dimensionalities), not a list.  I know it doesn't matter, the way this is read back... but this will make alternative loading approaches more easy to implement.